daemon/swaps/40swap.go

```diff
@@ -24,14 +24,14 @@ func NewClient(endpoint string) (*Client, error) {
 	}, nil
 }
 
-func chainToDtoChain(chain models.Chain) (api.SwapOutRequestDtoChain, error) {
+func chainToDtoChain(chain models.Chain) (string, error) {
```

Comment:
I hate having to do this xD

Reply:
I know, it doesn't make sense that it uses two different structs for the exact same thing. I'll check if there is any other way to do it

Reply:
done